src/components/overlays/AlertDialog/AlertDialog.tsx

```diff
@@ -70,7 +72,7 @@ function AlertDialog(props: CubeAlertDialogProps, ref) {
           <Title>{title}</Title>
         </Header>
       ) : null}
-      {children ? <Content>{children}</Content> : null}
+      <Content>{content}</Content>
```

Comment:
Why so? It's possible that there is no content, just a header.

Reply:
In the current version, content is a required field and I see it reasonable since if we won't have this or title required, it makes it possible to render `<AlertDialog />` without any description which is a bad practice.